src/assets/css/screens/gameScreen.css

```diff
@@ -1,3 +1,5 @@
+@import url(https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css);
```

Comment:
Copy content of this url.
Create bootstrap.min.css in assets/css and paste url content there
Import this file in core/bootstrap.ts